src/vectorpartitions.jl

```diff
@@ -0,0 +1,124 @@
+
+export vector_partitions
+
+function vector_partitions(vector::Vector{Int64},min=zeros(Integer,length(vector)))
+
+    #   Creates all vector partitions of "vector" with all parts greater than
```

Comment:
Maybe this (and the similar comments in functions below) could be moved above the function as a [doc string](https://docs.julialang.org/en/v1/manual/documentation/#Writing-Documentation)?